src/Data/OrderDocument/LocalStorage/localStorageOrderDocumentDataClass.ts

```diff
@@ -1,7 +1,27 @@
+import { currentLanguage, load } from 'scrivito'
 import { provideLocalStorageDataClass } from '../../provideLocalStorageDataClass'
-import { DataClassAttributes } from '../../types'
+import { DataClassSchema } from '../../types'
 
-const attributes: DataClassAttributes = {}
+async function attributes(): Promise<DataClassSchema> {
+  const lang = await load(currentLanguage)
+
+  return {
+    documentId: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Dokument' : 'Document',
+        to: 'Document',
+      },
+    ],
+    orderId: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Auftrag' : 'Order',
```

Comment:
* [ ] FYI Maybe we should harmonize the titles in the app and in the backend. The app currently uses "Bestellung".

Reply:
Good point. I guess "Aufträge" is well established in PisaSales.

Here is a working copy to adjust the naming in the portal app: https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://pisa.scrivito-portal-app.pages.dev/d0a154d76edf2a7bd991fc658e700a1d/de/auftraege-31f54bc713bc6050?_scrivito_workspace_id=s9deb2f670bc4592&_scrivito_display_mode=view

While doing so I noticed two or three smaller issues with the linking and also with the overview page in english.

Reply:
@dcsaszar please review the working copy and publish it, if you like it.

Reply:
I agree with ChatGPT here, maybe discuss the wording with Andreas?:

> [...], your welcome message is generally clear, but it could be fine-tuned for clarity and flow in both languages:
> 
> English:
> 
> “Welcome to My Tynacoon! Here you can view our offers and your orders.”
> 
> 	•	“Quotes” could be interpreted differently depending on context, so if you’re referring to price offers, “offers” might be more precise.
> 
> German:
> 
> “Willkommen bei Mein Tynacoon! Hier können Sie unsere Angebote und Ihre Bestellungen einsehen.”
> 
> 	•	“Aufträge” might suggest formal assignments or contracts, whereas “Bestellungen” (orders) would be clearer if you’re referring to purchases or customer orders.
> 
> If “quotes” actually refers to price quotations, your original “Angebote” is appropriate for both.